prdoc/pr_7377.prdoc

```diff
@@ -0,0 +1,13 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Add missing events to nomination pool extrinsics
```

Comment:
Nit: you can mention for which extrinsics you are adding event for.

Reply:
I added that to the description, thank you for pointing it out.